### PR TITLE
Feature/958 more that 30 seconds

### PR DESCRIPTION
**Pull request by ifirmawan**
### TODO/DONE

- [x] Create new table: jobs
- [x] Handle background sync submission with jobs status to prevent multiple submissions
- [x] Handle foreground sync submission with jobs status to prevent multiple submissions

### Fix feedback

- [x] Clear all jobs when logged out
- [x] Delete the Job when it's finished

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206402700868352

### CODE REVIEW

File: app/src/database/crud/crud-jobs.js
Comment (dedenbangkit) [inline app/src/database/crud/crud-jobs.js:1]: Delete the Job when it's finished

File: app/App.js
Comment (dedenbangkit) [inline app/App.js:1]: Clear all jobs when logged out

**Review by dedenbangkit — CHANGES_REQUESTED** [review]

File: app/App.js
Comment (ifirmawan) [reply app/App.js:1]: Resolved with this commit: https://github.com/akvo/rtmis/pull/1047/commits/5e04f4c8e4396df773e7b67fc8facf7f2fe80fcd

File: app/src/database/crud/crud-jobs.js
Comment (ifirmawan) [reply app/src/database/crud/crud-jobs.js:1]: Resolved with this commit https://github.com/akvo/rtmis/pull/1047/commits/73abcae49c6420e4d58be5195ba6a291738faea9

**Pull request merged**